htdocs/fourn/class/fournisseur.facture.class.php

```diff
@@ -2481,6 +2481,38 @@ public function insert($notrigger=0)
             $this->db->rollback();
             return -2;
         }
-    }
-}
+    } 
+            /**
+     *  Mise a jour de l'objet ligne de commande en base
+     *
+     *  @return		int		<0 si ko, >0 si ok
+     */
+    function update_total()
```

Comment:
What is goal of this.
The function update_total exists in a parent table (ex: llx_facutre) to update denormalized fields of total that store total of child lines (ex: llx_facturedet).
What is goal of having a update_total to update field on same table. The update method already exists.